extra-requirements.txt

```diff
@@ -51,7 +51,7 @@ protobuf>=3.13.0:           core
 grpcio>=1.33.1:             core
 pyyaml>=5.3.1:              core
 tornado>=5.1.0:             core
-cookiecutter:               hub, devel, cicd
+cookiecutter:               hub, devel, cicd, test
```

Comment:
```suggestion
cookiecutter:               hub, devel, cicd
```

Reply:
I am adding tests that require these dependencies.

Reply:
If I remove this, it won't pass

Reply:
no, `cicd` is the correct tag.
`test` is all "testsuit"-dependent packages

Reply:
ah ok